src-docs/src/views/highlight_and_mark/highlight_and_mark_example.js

```diff
@@ -4,14 +4,19 @@ import { renderToHtml } from '../../services';
 
 import { GuideSectionTypes } from '../../components';
 
-import { EuiCode, EuiHighlight } from '../../../../src/components';
+import { EuiCode, EuiHighlight, EuiMark } from '../../../../src/components';
 
 import { Highlight } from './highlight';
+import { Mark } from './mark';
+
 const highlightSource = require('!!raw-loader!./highlight');
 const highlightHtml = renderToHtml(Highlight);
 
-export const HighlightExample = {
-  title: 'Highlight',
+const markSource = require('!!raw-loader!./mark');
+const markHtml = renderToHtml(Mark);
+
+export const HighlightAndMarkExample = {
+  title: 'Highlight and Mark',
```

Comment:
We keep all titles in sentence case

```suggestion
  title: 'Highlight and mark',
```